Sessionless Product Detail.md

```diff
@@ -0,0 +1,50 @@
+# Sessionless Product Details
+
+**Introduction & Base Definitions**
+
+
+Authentication protocols (auth protocols) are gatekeepers that verify the identity of a person to give them access to specific programs or data. They include a set of rules or procedures that govern how data is transmitted across electronic devices. The most common uses of auth protocols are passwords, but others include one-time-passwords, biometrics, and magic links. 
```

Comment:
Let's include a link to this page: https://www.okta.com/identity-101/identification-vs-authentication/#:~:text=Identification%20is%20the%20act%20of,access%20to%20systems%20or%20privileges.

Reply:
Updated with latest merge